examples/src/main/python/sql/datasource.py

```diff
@@ -69,6 +69,25 @@ def generic_file_source_options_example(spark):
     # +-------------+
     # $example off:load_with_path_glob_filter$
 
+    # $example on:load_with_modified_date_filter$
+    # Assume file in provided path were last modified today.
+    df = spark.read.load("examples/src/main/resources/dir1",
+                         format="parquet", modifiedDateFilter="2020-06-01T08:30:00")
```

Comment:
Should we specify the timezone here? What should the default timezone be?
@cloud-fan @MaxGekk

Reply:
I wonder if we should be defaulting to UTC unless this option below or a spark conf setting for timezone is specified.  If one is specified, interpret the timestamp according to that offset from UTC?

![image](https://user-images.githubusercontent.com/659214/86691815-08c45f00-bfbe-11ea-91e1-57605333aa45.png)

Reply:
Great question by the way..thank you.